### PR TITLE
[phel/en] Add phel.html

### PR DESCRIPTION
**Pull request by Chemaclass**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: phel.html.markdown [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+
+(reduce push [] '(3 2 1))
+# = (push (push (push [] 3) 2) 1)
+# => [3 2 1]
+
+# Functions
+#############
+
+# Use fn to create new functions. A function always returns its last statement.
+(fn [] "Hello World") # => :function
```
Comment (jenshaase) [inline phel.html.markdown:142]: Output in REPL is `<function>`

File: phel.html.markdown [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+
+# You need extra parens to call it
+((fn [] "Hello World")) # => "Hello World"
+
+# You can create a var using def
```
Comment (jenshaase) [inline phel.html.markdown:147]: Better: You can bind a value to a symbol using def for definition

PS: `var` is also used for variables

File: phel.html.markdown [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+(def x 1)
+x # => 1
+
+# Assign a function to a var
```
Comment (jenshaase) [inline phel.html.markdown:151]: Assign a function to a definition

File: phel.html.markdown [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+((set 1 2 3) 1) # => 1
+((set 1 2 3) 4) # => nil
+
+# There are more functions like: count, union, intersection, difference, etc.
+
+
+# Useful forms
+#############
+
+# Logic constructs in clojure are just macros, and look like everything else
```
Comment (jenshaase) [inline phel.html.markdown:243]: Don't know how deep you want to go into this. But `if` is not a macro. It is a special form ;) But I think this is out of scope for this introduction.

**Review by jenshaase — APPROVED** [review]

File: phel.html.markdown [hunk 5]
```diff
@@ -0,0 +1,337 @@
...
+## Features
+- Built on PHP's ecosystem
+- Good error reporting
+- Persistent Datastructures (Lists, Vectors, Maps and Sets)
+- Macros
+- Recursive functions
+- Powerful but simple Syntax
+- REPL
+
+```phel
```
Comment (adambard) [inline phel.html.markdown:20]: I think pygments will probably choke on this; know any other lisps that use `#` for comments that you could sub in?

Comment (Chemaclass) [reply phel.html.markdown:20]: CC: @jenshaase. Do you know something that could help us with that question?

> Or maybe we should really consider allowing `;` as comments as well in phel, so we can easily use the `clojure`/`lisp` syntax colouring everywhere 😂

Comment (jenshaase) [reply phel.html.markdown:20]: There are a few lisp that use `#` for comments. However, in `pygments` there is currently only `newlisp` that supports this. I took a look at the demo (https://pygments.org/demo/) and the highlighting looks ok to me. There is only a small problem with backslashes

Comment (Chemaclass) [reply phel.html.markdown:20]: Right, everything looks good using `newlisp` except the backslashes... but I think it would be fine for now, don't you think? Do you want me to change `phel` to `newlisp` here (line 20)? I am fine with that, unless you have something different in your mind.
Screenshot 2022-01-19 at 09 40 13

**Comment by adambard** [comment]
Alright, let's give it a shot!

**Comment by jenshaase** [comment]
@adambard Thanks a lot for your support and merging this PR. May I ask if there is problem with the deployment? I noticed that this and other recent PRs didn't appear on the website. If there is anything I can do to fix this please let me know.

**Comment by Chemaclass** [comment]
@adambard I think either master needs to be deployed manually, or there is a problem with the auto-deployment of master to production, because I cannot find Phel in https://learnxinyminutes.com/ 🤔
